packages/bot-web-ui/src/components/dashboard/tour-slider.tsx

```diff
@@ -0,0 +1,144 @@
+
+    const onCloseTour = () => {
+        setOnBoardTourRunState(false);
+        setTourActive(false);
+    };
+
+    const onTourEnd = () => {
+        if (step === 7) {
+            onCloseTour();
+            setHasTourEnded(true);
+            setTourDialogVisibility(true);
+        }
+    };
+
+    const onChange = (param: string) => {
+        if (param === 'inc' && step < Object.keys(DBOT_ONBOARDING_MOBILE).length) setStep(step + 1);
+        else if (param === 'dec' && step > 1) setStep(step - 1);
+    };
+
+    React.useEffect(() => {
+        Object.values(DBOT_ONBOARDING_MOBILE).forEach(data => {
+            if (data.key === step) {
+                setContent(data?.content);
+                setheader(data?.header);
+                setimg(data?.img);
+            }
+        });
+    }, [step]);
+
+    return (
+        <>
+            <div className='dbot-slider'>
+                <div className='dbot-slider__navbar'>
+                    <Text weight='less-prominent' line_height='s' size='xxs'>{`${step}/7`}</Text>
+                    <Text weight='less-prominent' line_height='s' size='xxs' onClick={onCloseTour}>
+                        Exit Tour
+                    </Text>
+                </div>
+                {slider_header && (
+                    <Text
+                        weight='bold'
+                        align='center'
+                        className='dbot-slider__title'
+                        as='span'
+                        line_height='s'
+                        size='xs'
+                    >
+                        {slider_header}
+                    </Text>
+                )}
+                {slider_image && (
+                    <div className='dbot-slider__image'>
+                        <img src={slider_image} />
+                    </div>
+                )}
+                {slider_content && (
+                    <Text align='center' className='dbot-slider__content' as='span' line_height='s' size='xxs'>
+                        {slider_content}
+                    </Text>
+                )}
+                <div className='dbot-slider__status'>
+                    <div className='dbot-slider__progress-bar'>
+                        <ProgressBarOnboarding
+                            step={step}
+                            amount_of_steps={Object.keys(DBOT_ONBOARDING_MOBILE)}
+                            setStep={setStep}
+                        />
+                    </div>
+                    <div className='dbot-slider__button-group'>
+                        {step !== 1 && (
+                            <TourButton
+                                type='default'
+                                onClick={() => {
+                                    onChange('dec');
+                                }}
+                            >
+                                {localize('Previous')}
+                            </TourButton>
```

Comment:
I guess  you can try button name as prop and handle `localize() ` inside `onChange` method , it would be more readable and concise.

Reply:
done ✅